OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -3023,6 +3023,14 @@ public static DreamValue NativeProc_view(NativeProc.Bundle bundle, DreamObject?
         if (center is null)
             return new(view);
 
+        if (center.TryGetVariable("contents", out var centerContents) && centerContents.TryGetValueAsDreamList(out var centerContentsList)) {
+            foreach (var content in centerContentsList.GetValues()) {
+                view.AddValue(content);
+            }
```

Comment:
```suggestion
             view.AddRange(centerContentsList.GetValues());
```
pretty sure this can just be an `AddRange()`?

Reply:
`view` is a DreamList which does not have any range-adding methods. Closest would be `OperatorAppend()` but that should probably be only be used by DM.